source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -40,6 +40,14 @@ bool CLSBeamlineStatus::isOff() const
 	}
 }
 
+AMControl * CLSBeamlineStatus::beamlineStatusPVControl()
+{
+	if (beamlineStatusPVControl_ )
+		return beamlineStatusPVControl_ ;
+	else
+		return this;
```

Comment:
Or just `return beamlineStatusPVControl_;` and have it initialized to 0 in the constructor.